docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,72 @@
+# Release 409 (1 Mar 2023)
+
+## General
+
+* Add support for dropping fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Remove the `distributed-index-joins-enabled` configuration property and
+  related functionality. ({issue}`15375`)
+* Fix failure when using non-comparable and non-sortable fields as part of a
+  `VALUES` expression. ({issue}`)
+
+## BigQuery connector
+
+* Fix failure when non-lowercase column names exist in the `query` table
+  function. ({issue}`16075`)
+
+## ClickHouse connector
+
+* Remove support for the `ru.yandex.clickhouse.ClickHouseDriver` legacy JDBC
```

Comment:
Isnt the driver embedded .. so what is the user facing consequence of that? New version requirement maybe?

Reply:
`clickhouse.legacy-driver` became defunct config in this change.